### DISCUSSION TITLE
ipv6 and userland proxy disabled

### DISCUSSION THREAD

**Discussion by hrolofs** [nextcloud/all-in-one#2950] [Questions] [answered]
Hello together,

i have a nextclound 27 instance running behind traefik and ipv6 enabled and userland proxy disabled and this works well. Now i'm starting with first tests of nextcloud-aio.

And there i have one problem running it with ipv6 and "userland-proxy": false in the docker daemon.json

nextcloud-aio insallation:

- pv4 only: 
works

- ipv6 enabled (without userland-proxy configuration": 
works 

- ipv6 enabled and "userland-proxy": false: 
error during insallation, after submit domain naime i got the following error

Domain does not point to this server or the reverse proxy is not configured correctly. See the mastercontainer logs for more details. ('sudo docker logs -f nextcloud-aio-mastercontainer')

The "normal" nextcloud works with and problems and disabled userland proxy.

Is this an known issue?

Regards

**Comment by Zoey2936** [comment, marked as answer]
why do you want to disable userland-proxy?
